src/database.cpp

```diff
@@ -1,11 +1,23 @@
 #include "database.hpp"
 
-void Database::compact()
+Database::Database()
+{
+    createRTree = false;
```

Comment:
You can set this already in the initializer list like you do below in the second constructor